wire-ios/Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController.swift

```diff
@@ -411,8 +411,11 @@ final class ConversationViewController: UIViewController {
                     assertionFailure("mlsService is missing")
                     return
                 }
-
-                let resolver = OneOnOneResolver(migrator: OneOnOneMigrator(mlsService: mlsService))
+                let featureRepository = FeatureRepository(context: syncContext)
+                let getMLSFeatureUseCase = GetMLSFeatureUseCase(featureRepository: featureRepository)
```

Comment:
I see this a lot... perhaps we should perform it once and store it (such as on the user session) for future use.

Reply:
I don't use GetMLSFeatureUseCase anymore.